File: Java/SumOfFirstN.java
//Java program to calculate sum of first N natural numbers.

import java.util.Scanner;

class SumOfFirstN{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter a number : ");
        String x = sc.next();
        int n = Integer.parseInt(x);

        int sum = 0;
        for(int i=1; i<=n; i++){
            sum += i;
        }
        System.out.println("The sum of first "+n+" natural numbers is = "+sum);
    }
}

File: Python/sumoffirstn.py
# Python program to calculate sum of first N natural numbers.

n = int(input('Enter the number : '))
sum = 0

for i in range(1, n+1):
    # print(str(i))
    sum = sum + i

print('Sum of first '+str(n)+' natural numbers is = '+str(sum))

# Another way of printing the result using f string or string literals
# print(f'Sum of first {n} natural numbers is = {sum}')